cmis_stock_production_lot/models/stock_production_lot.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+
+
+from odoo import models, api
+from odoo.addons.cmis_field import fields
+
+
+class StockProductionLot(models.Model):
+    _inherit = 'stock.production.lot'
+
+    cmis_folder = fields.CmisFolder(
+        copy=False,
+        create_name_get='_cmis_create_name_get',
+        create_parent_get='_cmis_create_parent_get',
+        oldname='cmis_objectid')
+
+    @api.multi
+    def _cmis_create_name_get(self, field, backend):
+        ret = {}
```

Comment:
IMO, This method will return the same result as the default one provided by the cmis_folder field when you don't specify the 'create_name_get' parameter https://github.com/acsone/alfodoo/blob/10.0/cmis_field/fields/cmis_folder.py#L44. You should remove it
You can remove it.